src/dashboard_compiler/panels/charts/gauge/compile.py

```
@@ -0,0 +1,172 @@
+from dashboard_compiler.panels.charts.gauge.view import (
+    KbnGaugeStateVisualizationLayer,
+    KbnGaugeVisualizationState,
+)
+from dashboard_compiler.panels.charts.lens.metrics.compile import compile_lens_metric
+from dashboard_compiler.shared.config import random_id_generator
+
+if TYPE_CHECKING:
+    from dashboard_compiler.panels.charts.lens.columns.view import KbnLensColumnTypes, KbnLensMetricColumnTypes
+
+
+def compile_gauge_chart_visualization_state(  # noqa: PLR0913
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)